### DISCUSSION TITLE
Error: Can't find stylesheet to import - Issue with Chirpy Theme on AWS Lightsail Ubuntu 24.04

### DISCUSSION THREAD

**Discussion by padeume** [cotes2020/jekyll-theme-chirpy#2116] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Built from `jekyll-theme-chirpy`

### Description

1s
Run bundle exec jekyll b -d "_site"
Configuration file: /home/runner/work/padeume.github.io/padeume.github.io/_config.yml
 Theme Config file: /home/runner/work/padeume.github.io/padeume.github.io/_config.yml
            Source: /home/runner/work/padeume.github.io/padeume.github.io
       Destination: /home/runner/work/padeume.github.io/padeume.github.io/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
Error: Can't find stylesheet to import.

### Operations you have already tried

AWS Lightsail ubuntu 24.04
ruby 3.2.2

### Anything else?

Hello,

I am experiencing an issue while setting up the Chirpy theme on my Jekyll blog. Here's my setup:

- **Environment**:
  - **Operating System**: AWS Lightsail Ubuntu 24.04
  - **Ruby Version**: 3.2.2
  - **Jekyll Version**: (please include your Jekyll version here, e.g., `4.x`)

- **What I'm Trying to Do**:
  I am running a Jekyll blog on my local Ubuntu server, using it as the backend for a GitHub Pages blog. The default Jekyll theme works fine, but when I attempt to switch to the Chirpy theme, I encounter the following error during the build process:

  ```
  Error: Can't find stylesheet to import.
  ```

- **Steps I Followed**:
  1. Installed Ruby 3.2.2 and Jekyll on my Ubuntu server.
  2. Cloned the Chirpy theme repository and added the necessary configurations (`_config.yml`) to my project.
  3. Ran the build command:
     ```bash
     bundle exec jekyll b -d "_site"
     ```
     However, the build fails with the above error.

- **Troubleshooting Done**:
  1. Verified that all Sass files and their paths are correctly referenced.
  2. Replaced `@import` with `@use` in the `main.scss` file to comply with Dart Sass standards, but the issue persists.
  3. Cleared Jekyll cache and `_site` directory before retrying the build.
  4. Updated all dependencies (`bundle update`) and confirmed the `jekyll-sass-converter` version is up to date.
  5. Checked for potential conflicts between files in the `_tabs`, `assets`, and root directories, but couldn't identify the root cause.

- **Expected Outcome**:
  The Jekyll build process should complete successfully, generating the necessary files in the `_site` directory without errors.

- **Actual Outcome**:
  The build fails with the following error:
  ```
  Error: Can't find stylesheet to import.
  ```

I would greatly appreciate any guidance on resolving this issue. Thank you in advance for your support!

Best regards,  
padeume.

---

### Additional Suggestions:
1. Attach any relevant logs or directory structures (like `_sass`, `assets`, or `main.scss`) if you're comfortable sharing them.
2. Make sure to specify the exact Chirpy version you're using (e.g., `gem "jekyll-theme-chirpy", "~> 5.x"` or a Git clone).

This will make your issue clear, polite, and professional! 😊

**Comment by passplease** [comment]
Do like #1809
